WordPress-VIP-Go/ruleset.xml

```diff
@@ -104,6 +104,10 @@
 
 
 	<!-- Warnings and other things -->
+	<rule ref="WordPressVIPMinimum.Filters.RestrictedFilter.UploadMimes">
+		<type>warning</type>
+		<severity>10</severity>
```

Comment:
Why the choice of `10` here? Not saying it's wrong, just looking for a rationale between the items that are `7` and `10` here.